share/src/main/webapp/components/people-finder/people-finder.js

```diff
@@ -289,7 +289,18 @@
                      me.followingAllowed = true;
                   },
                   scope: this
-               }
+               },
+                failureCallback:
+                    {
+                        fn: function(response)
+                        {
+                           /* MNT-18453 - do nothing but report it */
+                            var retStatusCode = response.serverResponse.status;
+                            console.warn("people-finder: Unable to get a suitable response for users the current user is following, returned status was: " + retStatusCode);
```

Comment:
`console.warn()` isn't used directly by Share as there are some browsers (older MSIE's) that don't support it. Having the `Alfresco.logger.warn()` is enough here.